Tasks/CargoAuthenticateV0/_buildConfigs/Node20/package.json

```diff
@@ -27,8 +27,8 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^20.3.1",
     "@types/uuid": "^8.3.0",
-    "azure-pipelines-task-lib": "4.1.0",
-    "azure-pipelines-tasks-artifacts-common": "^2.241.0",
+    "azure-pipelines-task-lib": "4.13.0",
```

Comment:
We should have a range here ^4.13.0 instead of pinning to a single version

Reply:
Fixed!